### ISSUE TITLE
will you release a pretrained checkpoint of B/2 model？

### ISSUE THREAD

**Issue by aaab8b** [sail-sg/MDT#45]
Thanks for the wonderful work again and I still wants to do some experiments on the small models like B/2 or S/2 due to my limited resources. Will you guys release a pretrained checkpoint of B/2 model or S/2 model?

**Comment by gasvn** [comment]
Sorry I no longer have these checkpoints since I no longer have the access to the clusters used to train our model.

**Comment by aaab8b** [comment]
> Sorry I no longer have these checkpoints since I no longer have the access to the clusters used to train our model.

I use the same 32x8 batchsize and check every single detail and cannot get the same fid result for training my s/2 and b/2 models. The performance drops about ~20 fid score on my s/2 model. And the only thing I'm not certain about is your dynamic ratio. the code its from 0.3 to 0.5. but in your paper it's 0.3 to 0.05. which is correct?

**Comment by gasvn** [comment]
The mask ratio is from 0.3 to 0.5. Did you check if you are using the cfg for testing?
